src/store/api/serializers.py

```diff
@@ -14,31 +26,43 @@ class ProductSearchSerializer(serializers.Serializer):
     created_at = serializers.DateTimeField(read_only=True, format="%Y-%m-%d %H:%M")
     updated_at = serializers.DateTimeField(read_only=True, format="%Y-%m-%d %H:%M")
 
-    def get_discounted_price(self, obj: app_models.Product):
-        if obj.discount:
-            return obj.price - (obj.price * obj.discount / 100)
-        return None
-
     def to_representation(self, instance):
+        data = super().to_representation(instance)
+        return self.remove_discount_fields(data)
```

Comment:
why do you need to remove them?

Reply:
I wanted the client not to see the discount and the discounted price if there is None.